### PR TITLE
My Fixes

### PR DESCRIPTION
**Pull request by qzhsjz**
I'm so sorry for my attitude.
Sincerely to awmleer.

### CODE REVIEW

File: src/pages/lab-reserve-detail/lab-reserve-detail.component.ts [hunk 1]
```diff
@@ -45,42 +45,29 @@ export class LabReserveDetailComponent implements OnInit, DoCheck {
...
     await this.labSvc.approvingReservation(reservationid);
     this.messageSvc.success('已同意该请求');
-    this.router.navigate(['currLab-reserve', 'labs', this.currLab.id]);
+    this.router.navigate(['lab-reserve', 'labs', this.currLab.id]);
     this.reservationListUpdate();
   }
 
   async rejectingReservation(reservationid: number) {
     await this.labSvc.rejectingReservation(reservationid);
     this.messageSvc.success('已抨击该请求');
```
Comment (awmleer) [inline src/pages/lab-reserve-detail/lab-reserve-detail.component.ts]: 请使用正常提示语

**Comment by qzhsjz** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        卧槽居然没改过来
        
    
        
            
        
    
    
        
        

        
    

    On 9/15/2018 17:30，卡晨<notifications@github.com> wrote： 


@awmleer commented on this pull request.



In src/pages/lab-reserve-detail/lab-reserve-detail.component.ts:
>    }
 
   async approvingReservation(reservationid: number) {
     await this.labSvc.approvingReservation(reservationid);
     this.messageSvc.success('已同意该请求');
-    this.router.navigate(['currLab-reserve', 'labs', this.currLab.id]);
+    this.router.navigate(['lab-reserve', 'labs', this.currLab.id]);
     this.reservationListUpdate();
   }
 
   async rejectingReservation(reservationid: number) {
     await this.labSvc.rejectingReservation(reservationid);
     this.messageSvc.success('已抨击该请求');

请使用正常提示语

File: src/pages/lab-reserve-detail/lab-reserve-detail.component.html [hunk 2]
```diff
@@ -16,13 +16,13 @@ <h3>基本信息</h3>
     描述：{{currLab.description}}<br>
   </p>
   <h3>预约情况</h3>
-  <nz-date-picker [nzFormat]="dateFormat" [(ngModel)]="startDate"></nz-date-picker>
+  <nz-date-picker [nzFormat]="dateFormat" [(ngModel)]="startDate" (ngModelChange)="reservationListUpdate()"></nz-date-picker>
```
Comment (awmleer) [inline src/pages/lab-reserve-detail/lab-reserve-detail.component.html]: `[(ngModel)]`这里应该是`[ngModel]`吧？

**Review by awmleer — CHANGES_REQUESTED** [review]
有一些细节需要完善。。

**Comment by qzhsjz** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        不，如果用[ngModel]无法获取输入的值，只能绑定一个值和一个函数了。
        
    
        
            
        
    
    
        
        

        
    

    On 9/15/2018 17:31，卡晨<notifications@github.com> wrote： 


@awmleer commented on this pull request.



In src/pages/lab-reserve-detail/lab-reserve-detail.component.html:
> @@ -16,13 +16,13 @@ <h3>基本信息</h3>
     描述：{{currLab.description}}<br>
   </p>
   <h3>预约情况</h3>
-  <nz-date-picker [nzFormat]="dateFormat" [(ngModel)]="startDate"></nz-date-picker>
+  <nz-date-picker [nzFormat]="dateFormat" [(ngModel)]="startDate" (ngModelChange)="reservationListUpdate()"></nz-date-picker>

[(ngModel)]这里应该是[ngModel]吧？

**Review by awmleer — APPROVED** [review]
Nice job!

**Pull request merged**